examples/nodejs/aws/eventbridge/webapp/src/App.tsx

```diff
@@ -220,6 +223,12 @@ const App = () => {
           <div className="bg-white p-4 rounded-lg shadow-md flex-1 space-y-3">
             <div className={"flex flex-row space-x-2 text-sm items-center"}>
               <h1 className="font-bold text-teal-700">Published Messages: </h1>
+              <div>
+                When DDB items are updated or deleted, EventBridge sends messages to the Momento Topic. Those messages
+                will show up here, because we've subscribed to the topic using the code below.
+              </div>
```

Comment:
I added a little more text to help explain the form widgets more. I didn't really know how to style them... they could probably use a bit of love from you when you get a chance :)

Reply:
feel free to add a commit to the PR or we can do it in a follow-up PR